spec/services/noko_client_spec.rb

```diff
@@ -66,20 +57,22 @@
           double("Noko::Record", link: noko_links)
         end
 
-        before do
-          allow(noko_result).to(
-            receive(:each).and_yield(noko_entry).and_yield(noko_entry_2))
-
-          allow(described_class.client).to(
-            receive(:get_entries).and_return(noko_result).twice)
-        end
-
         it "saves all entries in every page" do
           expect do
             described_class.import_entries(start_date, end_date)
           end.to change(Entry, :count).by(2)
         end
       end
+
+      context "when importing entries" do
+        let(:start_date) { Time.now.beginning_of_week.to_date - 1.week }
+
+        it "saves entries from the past two weeks" do
+          expect do
+            described_class.import_entries(start_date, end_date)
+          end.to change(Entry, :count).by(3)
+        end
```

Comment:
@cleicar I'm having a hard time understanding how your change is actually solving the problem that we are seeing. 

How is this going to help with people submitting this week entries for the past week? 

I see that the rake task remains unchanged: https://github.com/ombulabs/pecas/blob/cdf659ad519f7f474fdb307f8e365bcda56eacf8/lib/tasks/importer.rake#L11-L21

Wouldn't we need to make the start_date further back in that rake task?

Reply:
Hey @etagwerker the purpose of my changes is to use a bigger interval of days when importing the entries. 

The importer task was already updated in my previous PR: https://github.com/ombulabs/pecas/pull/54/files#diff-3ad61c686e56f51add5f30435601d020R12

But after that we realized that the service wasn’t getting all the entries from all pages, so this is the goal for this PR.

Reply:
@cleicar  makes sense now. Thanks!